chugbot/.platform/hooks/postdeploy/00_get_certificate.sh

```diff
@@ -2,13 +2,15 @@
 
 ENV=$(/opt/elasticbeanstalk/bin/get-config container -k environment_name)
 
-if [[ $ENV == "Chugbotwi-env" ]]; then    
+if [[ $ENV == "Chugbotwi-env" ]]; then
   DOMAIN="wi.campramahchug.org"
 elif [[ $ENV == "Chugbotne-env" ]]; then
   DOMAIN="ne.campramahchug.org"
 elif [[ $ENV == "Chugbotdc-env" ]]; then
   DOMAIN="dc.campramahchug.org"
-else   
+elif [[ $ENV == "Chugbotrb-env" ]]; then
```

Comment:
I forgot to add this before!

Reply:
Ah, great catch!

Would it be possible to add this step to the README.md instructions on setting up a new instance? That file is in the top-level directory. If you've hit your max hours, I can do that.